aggregator/aggregator.py

```
@@ -123,14 +123,10 @@ def calculate_age_group(birth_date):
         if not birth_date:
             return "Unknown"
         age = (datetime.now() - datetime.strptime(birth_date, "%Y-%m-%d")).days // 365
-        if age < 2:
-            return "0-2 years"
-        elif age < 5:
-            return "3-5 years"
-        elif age < 18:
-            return "6-17 years"
+        if age < 1:
+            return "1 year"
```

I think this should be a less than or equal to, not a strict less than.

Worth noting it's possible this might hide errors, if `datetime.strptime(birth_date, "%Y-%m-%d")` is some how ahead of `datetime.now()` (future date incorrectly set in FHIR, or maybe the aggregator container's clock is incorrectly set) resulting in a negative number of days, they'd be accepted and returned as a "1 year" old. Shouldn't happen, but might be worth a check to at least log if that somehow occurs (and maybe rejecting the record or convert it to an "Unknown" age).

---

Also just want to double check, was the explicit ask to group <1 year olds and 1 year olds together? Might still be worth having a separate `<1 year` category. Just my opinion there.

---

Excellent point @Stephen-ONeil ! Fixed